pkg/discovery/discovery.go

```diff
@@ -0,0 +1,34 @@
+package discovery
+
+import "path/filepath"
+
+func getInferredManifestFilePath(cwd string) string {
```

Comment:
if I use a different name than the ones in [this list](https://github.com/okteto/okteto/blob/master/pkg/discovery/manifest.go#L24) I'm still ignoring my okteto manifest. To test it:
- create okteto manifest with name `okteto-test.yml`
- add to `.oktetodeployignore` an entry for `okteto-test.yml`

Should we infer the manifest base on the content of the file (maybe checking if yaml file contains main keys as `dev`, `build`or `deploy`) instead rely on the name? Following the principle that my manifest can have any name respecting the accepted format I think it would be optimal.

Reply:
when we have a custom name for the manifest, we have to pass the -f flag, as the actual infer of the manifest is done with this lists of files.

As far as i know this would change behavour and is not scope of this PR to change the infer manifest behavour